src/common/types/store-data/unified-data-interface.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// this is similar to `TestEngine` interface from axe-core
+export interface ScanEngine {
+    name: string;
+    version: string;
+}
+
+export interface OSData {
+    name: string;
+    osVersion?: string;
+    userAgent?: string;
+}
+
+export interface ViewPortData {
+    platformHeight?: number;
+    platformWidth?: number;
+}
+
+export interface PlatformData {
+    osInfo: OSData;
+    viewPortInfo: ViewPortData;
+}
+
+export interface ToolData {
```

Comment:
This appears to be a wrapper for the ScanEngine interface. Why not just use the ScanEngine interface?

Reply:
This was to basically have more information about `Tool` which right now is limited to `ScanEngine` information; but might increase to include more properties.
Basically `ToolData` needs to be a much bigger umbrella than `ScanEngine`.